Documentation/project-docs/benchmarking.md

```diff
@@ -120,20 +120,19 @@ class Program
 
 7. Go to the corresponding CoreCLR source folder where the API you want to change exists (for example `coreclr\src\System.Private.CoreLib\shared\System`)
 8. Apply the optimization that you want to test
-9. Rebuild System.Private.Corelib with your change (optionally adding `-skipnative` if the change is isolated to managed code):
+9. Rebuild System.Private.Corelib with your change (optionally adding `-skipnative -skipbuildpackages` if the change is isolated to managed code):
 
-        C:\Projects\coreclr>build.cmd -release -skiptests -skipnative
+        C:\Projects\coreclr>build.cmd -release -skiptests -skipnative -skipbuildpackages
 
 10. For the next step, you have one of two options:
 
   - Rebuild given CoreFX part in Release:
 
           C:\Projects\corefx>build.cmd -c Release /p:CoreCLROverridePath=C:\Projects\coreclr\bin\Product\Windows_NT.x64.Release
 
-  - OR manually copy over the relevant files from within the root of the coreclr output folder to where `CoreRun.exe` lives within corefx (excluding the subdirectories). This ends up being much faster than the first option and if the only thing that changed is   `System.Private.Corelib.dll`, just copy that over:
-
-          Copy from: C:\Projects\coreclr\bin\Product\Windows_NT.x64.Release
-          To: C:\Projects\corefx\artifacts\bin\testhost\netcoreapp-Windows_NT-Release-x64\shared\Microsoft.NETCore.App\9.9.9\
+  - Force refresh of CoreCLR hard-link copy(this ends up being much faster than the first option):
+   
+          C:\Projects\corefx>build.cmd -restore /p:CoreCLROverridePath=C:\Projects\coreclr\bin\Product\Windows_NT.x64.Release /p:ConfigurationGroup=Release
```

Comment:
Use `-release` instead of `/p:ConfigurationGroup=Release` ?

Reply:
```
C:\git\corefx (doc -> origin)
λ build.cmd -restore -release /p:CoreCLROverridePath=C:\git\coreclrupstream\bin\Product\Windows_NT.x64.Release
MSBUILD : error MSB1001: Unknown switch.
Switch: -release

For switch syntax, type "MSBuild /help"
Build failed.
```

Reply:
`-c Release` please :)

Reply:
ooo yes...too used to 😄

Reply:
done